api/idp/api.go

```diff
@@ -28,8 +28,8 @@ type IdentityProviderRepresentation struct {
 }
 
 type HrdSettingModel struct {
-	IPRangesList string `json:"ipRangesList"`
-	Priority     int    `json:"priority"`
+	IPRangesList *string `json:"ipRangesList"`
+	Priority     *int    `json:"priority"`
```

Comment:
IPRangesList and DomainsList should be optional but why Priority is not mandatory?